runtime/themes/onelight.toml

```diff
@@ -79,7 +79,7 @@
 "ui.selection" = { bg = "light-white" }
 "ui.selection.primary" = { bg = "light-white", modifiers = ["dim"] }
 
-"ui.virtual" = { fg = "light-white"}
+"ui.virtual.whitespace" = { fg = "light-white"}
 "ui.virtual.ruler" = { bg = "light-white" }
 "ui.virtual.whitespace" = {fg = "light-white" }
```

Comment:
looks like this one already makes the distinction between ruler and whitespace

Reply:
Ops, good catch. Should I just get rid of `"ui.virtual"` for this one?

Reply:
Hmm I'd say let's leave it. It might be a good default for scopes in the future

Reply:
All right!